crates/turbopack/src/module_options/module_options_context.rs

```diff
@@ -187,11 +199,15 @@ pub struct ModuleOptionsContext {
     pub enable_mdx_rs: bool,
     #[serde(default)]
     pub preset_env_versions: Option<EnvironmentVc>,
+    #[deprecated(note = "use custom_ecma_transform_plugins instead")]
     #[serde(default)]
     pub custom_ecmascript_app_transforms: Vec<EcmascriptInputTransform>,
+    #[deprecated(note = "use custom_ecma_transform_plugins instead")]
     #[serde(default)]
     pub custom_ecmascript_transforms: Vec<EcmascriptInputTransform>,
     #[serde(default)]
+    pub custom_ecma_transform_plugins: Option<CustomEcmascriptTransformPluginsVc>,
```

Comment:
Nit: Just call these `custom_transform_plugins`?

Reply:
I expect we'll have things like css_transform later, based on experiences with swc's transform plugin.